packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from 'terra-section-header';
+import Toggle from 'terra-toggle';
+
+const VALID_LEVELS = [1, 2, 3, 4, 5, 6];
+
+const propTypes = {
+  /**
+   * Content in the body of the toggle section header component that will be expanded or hidden.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text to be displayed in the section header.
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, or 6.
+   */
+  level: PropTypes.oneOf(VALID_LEVELS),
+  /**
+   * Callback function triggered when toggle-section-header is closed.
+   */
+  onClose: PropTypes.func,
+  /**
+   * Callback function triggered when toggle-section-header is opened.
+   */
+  onOpen: PropTypes.func,
+  /**
+   * Used to set props and HTML attributes on the underlying section-header.
+   */
+  sectionHeaderAttrs: PropTypes.object,
+  /**
+   * Sets the toggle-section-header to be animated when it is opened or closed.
+   */
+  isAnimated: PropTypes.bool,
+  /**
+   * Sets the toggle-section-header initial state to open.
+   */
+  isInitiallyOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  level: 2,
```

Comment:
Is there a definition somewhere that indicates what each heading level renders as?

Reply:
the `level` prop is done solely for semantic purposes to ensure that this _Header_ type component can be appropriately recognized for accessibility purposes. It has no associated styles or values that do anything other than [pass the level through](https://github.com/cerner/terra-core/blob/toggle-section-header/packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx#L98) to the underlying `SectionHeader` component. This is solely a state-wrapper around `terra-section-header` & `terra-toggle`, with additional `onClose` and `onOpen` callbacks that we've provided in order to keep the API consistent with `terra-toggle-button` with first-class props providing the ability to hook into the underlying components' props.

Reply:
Fair enough. We don't need to test already tested components. Didn't think about that :)